pkg/controller/logstorage/esmetrics/esmetrics_controller.go

```diff
@@ -106,6 +115,16 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 		}
 	}
 
+	k8sClient, err := kubernetes.NewForConfig(mgr.GetConfig())
+	if err != nil {
+		return fmt.Errorf("log-storage-esmetrics-controller failed to establish a connection to k8s: %w", err)
+	}
+
+	go utils.WaitToAddTierWatch(networkpolicy.TigeraComponentTierName, c, k8sClient, log, r.tierWatchReady)
+	go utils.WaitToAddNetworkPolicyWatches(c, k8sClient, log, []types.NamespacedName{
+		{Name: linseed.PolicyName, Namespace: render.ElasticsearchNamespace},
```

Comment:
Why do we need to watch the Linseed policy in the ES metrics controller?

Reply:
It should be es-metrics policy.